modules/opensearch/opensearch.install

```diff
@@ -20,10 +20,19 @@ function opensearch_migrate_settings() {
     'ting_recommendation_url' => 'opensearch_recommendation_url',
     'ting_enable_logging' => 'opensearch_enable_logging',
     'ting_cache_lifetime' => 'opensearch_cache_lifetime',
+    'ting_search_autocomplete_suggestion_url' => 'opensearch_search_autocomplete_suggestion_url',
+    'ting_search_autocomplete_method' => 'opensearch_search_autocomplete_method',
+    'ting_search_autocomplete_settings' => 'opensearch_search_autocomplete_settings',
   );
 
   array_walk($settings_map, function($new_var, $old_var) {
+    $settings_keep = array(
+      'ting_search_autocomplete_settings',
```

Comment:
This way we keep the entire `ting_search_autocomplete_settings`. We should only keep the two remaining values.